.circleci/config.yml

```diff
@@ -1,282 +1,27 @@
-                          export CYPRESS_CONTACT_DETAILS_ENDPOINT=$(aws ssm get-parameter --name /housing-tl/<<parameters.stage>>/contact-details-api-url --query Parameter.Value --output text) ;
-                          export CYPRESS_EQUALITY_DETAILS_ENDPOINT=$(aws ssm get-parameter --name /housing-tl/<<parameters.stage>>/equality-information-api-url --query Parameter.Value --output text) ;
-                          export CYPRESS_PERSON_ENDPOINT=$(aws ssm get-parameter --name /housing-tl/<<parameters.stage>>/person-api-url --query Parameter.Value --output text) ;
-                          export CYPRESS_TENURE_ENDPOINT=$(aws ssm get-parameter --name /housing-tl/<<parameters.stage>>/tenure-api-url --query Parameter.Value --output text) ;
-                          export CYPRESS_CAUTIONARY_ALERT_ENDPOINT=$(aws ssm get-parameter --name /housing-tl/<<parameters.stage>>/cautionary-alerts-api-url --query Parameter.Value --output text) ;
-                          export CYPRESS_ENVIRONMENT=<<parameters.stage>> ;
-                          export CYPRESS_AWS_SECRET_ACCESS_KEY=$(aws configure get aws_secret_access_key --profile default) ;
-                          export CYPRESS_AWS_ACCESS_KEY_ID=$(aws configure get aws_access_key_id --profile default) ;
-                          export CYPRESS_AWS_SESSION_TOKEN=$(aws configure get aws_session_token --profile default) ;
-                          echo "Running against $CYPRESS_ENVIRONMENT environment" ;
-                          npm install ;
-                          if [ <<parameters.trigger-context>> == "external" ] && [ <<parameters.stage>> == "production" ]
-                          then ./node_modules/.bin/cypress run --record --key 31a18a85-d0e5-4c16-a1d4-5f0eab6a7c58 -e TAGS='@Production and not @GoogleLighthouse and not @Accessibility and not @ignore and not @device' --parallel --group 'all tests'
-                          else ./node_modules/.bin/cypress run --record --key 31a18a85-d0e5-4c16-a1d4-5f0eab6a7c58 -e TAGS='not @GoogleLighthouse and not @Accessibility and not @ignore and not @device' --parallel --group 'all tests'
-                          fi
-                       
-            - store_artifacts:
-                path: /root/project/cypress/videos/
-                destination: artifact-file
+orbs:
+  continuation: circleci/continuation@0.2.0
 
 jobs:
-  assume-role-development:
-        executor: docker-python
-        steps:
-            - assume-role-and-persist-workspace:
-                  aws-account: $AWS_ACCOUNT_DEVELOPMENT
+  determine-parallelism:
+    docker:
+      - image: cimg/base:edge
+    resource_class: small
+    steps:
+      - checkout
+      - run:
+          name: Determine parallelism
+          command: |
+            PARALLELISM=$(find ./cypress/e2e -type f -name "*.feature" | wc -l)
```

Comment:
We have decided to move away from Gherkin due to the needless complexity it introduces.

As such, the `.feature` files won't stay around within this repository for long.